### DISCUSSION TITLE
Overwritten Config

### DISCUSSION THREAD

**Discussion by drickerby** [UltimaHoarder/UltimaScraper#962] [Ideas]
Why does every new release of this have to delete my config and make me set it up again?

Doesn't help that Google have changed Chrome so I can't find my ID any more. Please either stop the updates overwriting my config or make better instructions so we can find the right information after Google update shit.

**Comment by UltimaHoarder** [comment]
My instructions are as braindead as it can get.

**Comment by drickerby** [comment]
But they don't work when Google update shit. I press F12 and just get graphs, rather than what's in your screenshots. I'd love to be able to search my cookies for api2 or even ?app but I just don't have that option any more. I must be doing something wrong....

**Reply by drickerby** [reply]
I was able to reset my developer console settings and now see all the right things again. I dunno how it changed, but it's sorted now. 

The only issue I seem to have now is that I want to rip all the videos of one particular model, but it seems to be going through all the models regardless.
